target/linux/mpc85xx/config-5.10

```diff
@@ -35,20 +35,14 @@ CONFIG_COMPAT_32BIT_TIME=y
 # CONFIG_CORENET_GENERIC is not set
 # CONFIG_CPM2 is not set
 CONFIG_CPU_BIG_ENDIAN=y
-CONFIG_CRYPTO_AEAD=y
-CONFIG_CRYPTO_AEAD2=y
+CONFIG_CRC16=m
```

Comment:
=m? 

This is strange. It's either "not set"  or "=y" in the target's kernel configurations. It's not that this doesn't "work", but maybe not as intended (since without including the CRC16 module in some way, this can backfire.)

Reply:
bumped to 6.6 not relevant anymore